integrations/extensions/starter-kits/coveo/README.md

```diff
@@ -0,0 +1,110 @@
+- Go to `Variables > Created by you` and add `query_text`, `result_items` `link0`, `link1`, `link2`, `title0`, `title1`, `title2`, `snippet0`,`snippet1`, and `snippet2`.
+- Create a new action and put "Search" in "What does your customer say to start this interaction?".  Add step 1:
+    - Click the fX button to add a variable and add new session variable `query_text` and select "Expression" type and then put `input.text` or `input.original_text` as the expression.  The former will employ spelling correction to fix any detected spelling errors before sending the query, which can be helpful, but it can also be counterproductive if your documents include specialized terminology that is not in our dictionary (such as product names) so you can use `input.original_text` as the alternative in such cases.
+    - Optional: In "Assistant says", put `Searching for: ${query_text}`
+    - In "And then", select "Use an extension", select the extension you made back in step 2, and select the search endpoint and set the `q` parameter to the `query_text` session variable.
+
+![Setup Query](./assets/store_query_text.png)<br>
+![Create Extension](./assets/setup_extension.png)<br>
+
+- Click "New Step" and change "without conditions" to "with conditions" and set "Ran successfully" to "false".  Also set "And then" to "End the action".  Then add the following to the "Assistant says":
+  Sorry.  The search failed!  Please try again another time.
+
+![Search failed](./assets/search_failed.png)<br>
+
+
+- Add a "New Step", then:
+    - Change "without conditions" to "with conditions" and select "true" for "Ran successfully"
+    - Click the fX button to add variables and add all the following new session variables, replacing `step_123_result_1` with the actual variable name and selecting "Expression" each time. For context on why we do this and what these mean, see [Extensions Made Easy with Watson Assistant Starter Kits](https://medium.com/ibm-watson/extensions-made-easy-with-watson-assistant-starter-kits-6b177f624697):
+```
+link0 = ${step_123_result_1}.body.items.get(0).clickUri
+title0 = ${step_123_result_1}.body.items.get(0).title
+snippet0 = ${step_123_result_1}.body.items.get(0).excerpt
+link1 = ${step_123_result_1}.body.items.get(1).clickUri
+title1 = ${step_123_result_1}.body.items.get(1).title
+snippet1 = ${step_123_result_1}.body.items.get(1).excerpt
+link2 = ${step_123_result_1}.body.items.get(2).clickUri
+title2 = ${step_123_result_1}.body.items.get(2).title
+snippet2 = ${step_123_result_1}.body.items.get(2).excerpt
+```
+
+![Map variables to results](./assets/set_variables.png)<br>
+
+- Click "New Step" and also change "without conditions" to "with conditions" and select "Ran successfully" is "true" and "link0" is "defined".  Then add the following to the "Assistant says":
+
+```
+<a href="${link0}">${title0}</a>
```

Comment:
This should be:

```
<a href="${link0}" target="_blank">${title0}</a>
${snippet0}
```

(Here and for the others below).

The `target="_blank"` is needed to make sure that the link opens up in a new tab, which is generally what you want if the links are pointing to a page that does not have the bot on it, because otherwise when you click on the link, the bot vanishes because you just opened up a different page in the tab that was running the bot.